qiime2/plugins.py

```diff
@@ -98,5 +98,11 @@ def exec_module(self, module):
                 for key, value in actions.items():
                     setattr(module, key, value)
 
+    def inspect_formats(filter=None, semantic_type=None):
+        import qiime2.sdk
+        pm = qiime2.sdk.PluginManager()
+
+        return pm.get_formats(filter=None, semantic_type=None)
```

Comment:
Should this be:

```suggestion
        return pm.get_formats(filter, semantic_type)
```

?

Otherwise it is impossible to set the `filter` or `semantic_type`.